filter1.js

```diff
@@ -24,3 +24,6 @@ var isCool = function(person) {
 }
 
 // Your code goes here
+coolPeople = people.filter(isCool)
+console.log(coolPeople)
+//I didnt know how to ONLY list the names instead of the full arrays
```

Comment:
that's fine, the result you got is what I expected! `filter()` does just that -- it filters an array, but in order to dig deeper into the objects inside in the array you would have to use something like `map()` together with the `filter()`.